src/tags/control/ClassificationBase.js

```diff
@@ -40,6 +41,23 @@ const ClassificationBase = types.model('ClassificationBase', {
     };
   }).actions(self => {
     return {
+      validate() {
+        if (self.perregion) {
+          return self._validatePerRegion();
+        }
+        else if (self.peritem && isFF(FF_LSDV_4583)) {
+          return self._validatePerItem();
+        }
+        else {
```

Comment:
that's something new! we don't use such style for if-else and I thought we have this covered by linting. if not, we have to add it to the list of linting improvements